ruby/rails/security/audit/xss/avoid-redirect.yaml

```diff
@@ -0,0 +1,50 @@
+rules:
+- id: avoid-redirect
+  metadata:
+    owasp: 'A6: Security Misconfiguration'
+    cwe: 'CWE-276: Incorrect Default Permissions'
+    references:
+    - https://brakemanscanner.org/docs/warning_types/redirect/
+    category: security
+    technology:
+    - rails
+  message: |
+    When a redirect uses user input, a malicious user can spoof a website under a trusted URL or access restricted parts of a site. When using user-supplied values, sanitize the value before using it for the redirect.
+  languages: [ruby]
+  severity: WARNING
+  mode: taint
+  pattern-sources:
+  - pattern: params
+  - pattern: cookies
+  - pattern: request.env
+  - patterns:
```

Comment:
Did not copy the rule entirely, this may be wrong.

Reply:
hm I have no idea how to do that, willing to learn at some point but preferably not this week or next